src/main/java/Duke.java

```diff
@@ -2,33 +2,67 @@
 import java.util.Scanner;
 
 public class Duke {
+    public static final int MAX_SIZE = 100;
 
     public static void main(String[] args) {
-        Scanner myWord = new Scanner(System.in);
-        Task[] t = new Task[100];
+        Scanner mySentence = new Scanner(System.in);
+        Task[] t = new Task[MAX_SIZE];
         int ctr = 0;
         welcomeMessage();
```

Comment:
Shouldn't method be named as a verb? eg. showWelcomeMessage

Reply:
Can welcomeMessage() have an action before it? Such as printWelcomeMessage